utils/distortions.py

```diff
@@ -0,0 +1,791 @@
+    return y
+
+
+def white_noise_cc(x: torch.Tensor, var: float, clip: bool = True, rounds: bool = False) -> torch.Tensor:
+    """
+        Adds Gaussian white noise to an input image tensor in the YCbCr color space
+        and optionally applies clipping and rounding after converting back to RGB.
+
+        Args:
+        x : torch.Tensor
+            Input image tensor with shape `(C, H, W)` or `(N, C, H, W)` where `C` is the number of channels.
+            Pixel values should be in the range `[0, 1]`.
+        var : float
+            Variance of the Gaussian white noise to be added.
+        clip : bool, optional
+            If `True`, clips the resulting tensor to the range `[0, 1]`. Default is `True`.
+        rounds : bool, optional
+            If `True`, rounds the resulting tensor to the nearest 8-bit integer (simulates quantization). Default is `False`.
+
+        Returns:
+        torch.Tensor
+            Output tensor with added Gaussian noise, converted back to RGB, and optionally clipped or rounded.
+            The output shape matches the input shape.
+    """
+    noise = torch.randn(*x.size(), dtype=x.dtype) * math.sqrt(var)
+
+    ycbcr = kornia.color.rgb_to_ycbcr(x)
+    y = ycbcr + noise
+
+    y = kornia.color.ycbcr_to_rgb(y)
+
+    if clip and rounds:
+        y = torch.clip((y * 255.0).round(), 0, 255) / 255.
+    elif clip:
+        y = torch.clip(y, 0, 1)
+    elif rounds:
+        y = (y * 255.0).round() / 255.
+
+    return y
+
+
+def impulse_noise(x: torch.Tensor, d: float, s_vs_p: float = 0.5) -> torch.Tensor:
+    """
+      Adds impulse noise (salt-and-pepper noise) to an input image tensor.
+
+      Args:
+      x : torch.Tensor
+          Input image tensor with shape `(C, H, W)` where `C` is the number of channels.
+          Pixel values should be in the range `[0, 1]`.
+      d : float
+          Density of the impulse noise. This value determines the proportion of pixels
+          affected by salt-and-pepper noise. Should be in the range `[0, 1]`.
+      s_vs_p : float, optional
+          Salt-to-pepper ratio. Determines the proportion of affected pixels that become
+          "salt" (white, value = 1) vs "pepper" (black, value = 0). Default is `0.5` (equal salt and pepper).
+
+      Returns:
+      torch.Tensor
+          Output tensor with added salt-and-pepper noise. The shape matches the input shape.
+    """
+    num_sp = int(d * x.shape[0] * x.shape[1] * x.shape[2])
+
+    coords = np.concatenate((np.random.randint(0, 3, (num_sp, 1)),
+                             np.random.randint(0, x.shape[1], (num_sp, 1)),
+                             np.random.randint(0, x.shape[2], (num_sp, 1))), 1)
+
+    num_salt = int(s_vs_p * num_sp)
+
+    coords_salt = coords[:num_salt].transpose(1, 0)
+    coords_pepper = coords[num_salt:].transpose(1, 0)
+    x[*coords_salt] = 1
+    x[*coords_pepper] = 0
+
+    return x
+
+
+def multiplicative_noise(x: torch.Tensor, var: float) -> torch.Tensor:
```

Comment:
var fica parecendo de variavel
```suggestion
def multiplicative_noise(x: torch.Tensor, variance: float) -> torch.Tensor:
```